src/hidra/receiver/asapo_transfer.py

```diff
@@ -23,7 +25,8 @@ class Stopped(Exception):
 class TransferConfig:
     def __init__(self, signal_host, target_host, detector_id,
                  endpoint, beamline, default_data_source,
-                 token, target_dir='', n_threads=1, start_file_idx=1, beamtime='auto', target_port=50101,
+                 token_path='/gpfs/asapo/shared/beamline_tokens',
+                 target_dir='', n_threads=1, start_file_idx=1, beamtime='auto', target_port=50101,
```

Comment:
Maybe `target_port` could be deleted? Is there a use case for choosing a specific port?

Reply:
Removed.